packaging/src/conf/brooklyn.conf

```diff
@@ -0,0 +1,332 @@
+# brooklyn.location.jclouds.machineCreateAttempts = 3
+## many cloud machines don't have sufficient entropy for lots of encrypted networking, so fake it:
+# brooklyn.location.jclouds.installDevUrandom=true
+
+#################################### Locations ###########################################
+
+# Named locations appear in the web console. If using the command line, it may be as easy
+# to use the jclouds:<provider> locations. 
+
+## Example: AWS Virginia using Rightscale 6.3 64bit Centos AMI and Large Instances
+# brooklyn.location.named.AWS\ Virginia\ Large\ Centos = jclouds:aws-ec2:us-east-1
+# brooklyn.location.named.AWS\ Virginia\ Large\ Centos.imageId=us-east-1/ami-7d7bfc14
+# brooklyn.location.named.AWS\ Virginia\ Large\ Centos.user=root
+# brooklyn.location.named.AWS\ Virginia\ Large\ Centos.minRam=4096
+
+## Example: Rackspace US using Debian
+# brooklyn.location.named.Rackspace\ US\ -\ Debian = jclouds:cloudservers-us
+# brooklyn.location.named.Rackspace\ US\ -\ Debian.image-name-regex = Debian
+
+## Example: AWS Oregon with Meta Data
+# brooklyn.location.named.AWS\ EC2\ Oregon = jclouds:aws-ec2:us-west-2
+# brooklyn.location.named.AWS\ EC2\ Oregon.userMetadata = application=my-oregon-app,owner="Awesome Brooklyn User"
+
+## Example: AWS Ireland using Alternative Keys
+# brooklyn.location.named.AWS\ EC2\ Ireland\ Alt \Keys = jclouds:aws-ec2:eu-west-1
+# brooklyn.location.named.AWS\ EC2\ Ireland.private-key-file = ~/.ssh/alternative_rsa
+# brooklyn.location.named.AWS\ EC2\ Ireland.public-key-file = ~/.ssh/alternative_rsa.pub
+
+## List of jclouds Compute Locations
+##  jclouds is a multi-cloud library that Brooklyn uses to access many clouds.
+##  Supported clouds: http://www.jclouds.org/documentation/reference/supported-providers/ 
+##
+##  Comment out those that you don't need.
+##  Remember to define your identity and credential for each provider
+##  e.g: brooklyn.location.jclouds.aws-ec2.identity = ?
+##       brooklyn.location.jclouds.aws-ec2.credential = ?
+##       brooklyn.location.jclouds.cloudservers-us.identity = ?
+##       brooklyn.location.jclouds.cloudservers-us.credential = ?
+##
+
+# Note: For the convenience of new users, we have specified Ubuntu 12.04 LTS AMIs within
+# AWS, deploying to m1.smalls, minimum. Ubuntu requires user = ubuntu. 
+# Please change these to better suit your requirements.
+# (Note: If no parameters are set jclouds will pick AMIs at random, user=root, t1.micros.)
+#brooklyn.location.jclouds.aws-ec2.identity = YOURAPIKEY
+#brooklyn.location.jclouds.aws-ec2.credential = YOURSECRETKEY
+#brooklyn.location.jclouds.aws-ec2.minRam = 1600
+#brooklyn.location.jclouds.aws-ec2.user=brooklyn
+#brooklyn.location.named.AWS\ Virginia\ (us-east-1) = jclouds:aws-ec2:us-east-1
+#brooklyn.location.named.AWS\ Virginia\ (us-east-1).imageId = us-east-1/ami-d0f89fb9
+#brooklyn.location.named.AWS\ Virginia\ (us-east-1).user = ubuntu
```

Comment:
I think it would be better to shrink this to about two-three examples?
Others WDYT?